### DISCUSSION TITLE
Can Google bypass OpenVPN on Android phones?

### DISCUSSION THREAD

**Discussion by capoaira** [pivpn/pivpn#1804] [Q&A]
Hello, I installed pivpn on my raspi and I used it all the time. I also have installed pi-hole as add-blocker. Now I can see which device send requests to which domains.
Now, when I'm in my own Wi-Fi network, I see all request by my smartphone comes from my VPN-IP-address of my smartphone (10.7.90.4), except requests to www.google.com, they come from my the IP of my smartphone, not from my VPN. How is that possible? Can Google bypass VPN on Android phones? Or do I miss something?
When I use Chrome and type google.com it comes through my VPN but no requests that I didn't initiate myself.

**Comment by TinCanTech** [comment]
I have heard rumors that google may be taking covert action against ad-blocking.

As google are the primary developer for Android, it is theoretically possible that chrome could by-pass a VPN.

Microsoft already do exactly that for their Office product and probably many other products.
